### ISSUE TITLE
conda build for pythonocc-7.4.0

### ISSUE THREAD

**Issue by tpaviot** [tpaviot/pythonocc-core#704]

**Comment by rainman110** [comment]
Here's the already working template:
https://github.com/DLR-SC/tigl-conda/tree/master/python-occ-7x

**Comment by tpaviot** [comment]
@rainman110 your recent fix actually finds Python better than previous version (I've been monitoring travis build). I did not try your recipe, but I saw you managed to build packages for all oses. Congrat's, that's a tough work. you can open a PR and submit your recipe as a whole if you want.

**Comment by tpaviot** [comment]
I should hide swig warnings in the conda recipe, there are tons of output polluting the console

**Comment by rainman110** [comment]
Keep fingers crossed that we can now finally build windows.

In my recipe, I have warnings disabled.

Will make a PR!

**Comment by rainman110** [comment]
Wow... Windows builds now out of the box on my machine! 😄 

Next milestone achieved!!!

**Comment by rainman110** [comment]
I'll trigger a new CI build.

**Comment by rainman110** [comment]
@tpaviot I have seen that you are now playing with conda and travis and the opengl settings.

Implementing this correctly can be very difficult imho. The problem is, that if you build your conda package againt Ubuntu's libgl* it might not work under other linux distros. Typical errors are undefined references or version mismatches.

To workaround this, pythonocc should either link against conda gl packages or against CDT packages of conda that provide a compatibility layer around the system libraries:
You can read more about this here: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#host

Long story short: Don't install systems opengl dev libaries, use condas.

**Comment by tpaviot** [comment]
Are you sure "playing" is the right word ? :blush: I agree with you. I think 4im gonna play in another branch, there might be several dumb commits.

**Comment by tpaviot** [comment]
Fixed, see issue #711
